config/routes.rb

```diff
@@ -1,4 +1,13 @@
 Rails.application.routes.draw do
+  # get 'pages/hello'
+  # root 'pages#hello'
+  root 'users#index'
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:new, :create]
+      resources :likes, only: [:new, :create]
```

Comment:
- [ ] Kindly remove unnecessary routes from your `config/routes` file and make sure you added routes to handle only the 4 required URLs as requested in the project requirement.
![Screenshot from 2023-12-19 19-26-53](https://github.com/BHObserver/blog-app/assets/91142432/fe3b6266-e640-4058-9493-c40160641ac1)

**HIT:** Your code could look like this
```rb
# config/routes.rb

  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```